n3/index.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for N3 v0.8.1 
+// Project: https://github.com/RubenVerborgh/N3.js
+// Definitions by: Fred Eisele <https://github.com/phreed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../node/index.d.ts" />
+
+/**
+### 0.8.1 Changelog (https://github.com/RubenVerborgh/N3)
+
+#### TODO:
+
+Match samples from README.md
+
+- [x] _.first as an alias of _.head
+
+*/
+
+declare module "n3" {
```

Comment:
Since this definition consists only of `declare module`, just remove this wrapper. You'll also need to add `declare` in front of `namespace N3`.